uint8 groupMask
uint32 remainValidMillisecs
---
